backend/src/apiserver/template/argo_template.go

```diff
@@ -230,7 +230,7 @@ func ValidateWorkflow(template []byte) (*util.Workflow, error) {
 	if wf.Kind != argoK8sResource {
 		return nil, util.NewInvalidInputError("Unexpected resource type. Expected: %v. Received: %v", argoK8sResource, wf.Kind)
 	}
```

Comment:
@gmfrasca could you take a look at these argo code changes please?

Reply:
This is due to a breaking change in the Argo's API. `ValidateWorkflow` returns only `error` in the new version.

Reply:
lgtm